ui/src/components/Search/index.jsx

```diff
@@ -16,7 +16,7 @@ import { THEME } from '../../utils/constants';
     },
     '& $input': {
       transition: theme.transitions.create('width'),
-      width: 200,
+      width: 230,
```

Comment:
On mobile, we can no longer see the book icon (docs)

![Screen Shot 2019-10-08 at 12 32 29 PM](https://user-images.githubusercontent.com/3766511/66414657-dacad380-e9c7-11e9-8586-d48a3ece234c.png)

It used to be

![Screen Shot 2019-10-08 at 12 33 01 PM](https://user-images.githubusercontent.com/3766511/66414711-f03ffd80-e9c7-11e9-90bc-5c832caa9133.png)


What do you think we should do?

Reply:
We can keep the search box `width` as same 200 in mobile-view but can reduce the font-size. I think, this would work.